docker/etna-base/entrypoints/build.sh

```diff
@@ -73,3 +73,5 @@ fi
 
 echo 'for file in /app/*.completion; do source $file || true; done' >> /root/.bashrc
 echo 'export PATH="/app/bin:$PATH"' >> /root/.bashrc
+# Allow other users to use the root bash setup
+chmod -R 777 /root/
```

Comment:
This is safe -- the only real things in /root/ are the bash and completion objects anyways, which I want to make more easily available.